manager.py

```diff
@@ -78,6 +77,7 @@ def __init__(self,
                  datalog=None,
                  datalogflag=False,
                  protocol=DEFAULT_PROTOCOL,
+                 test = None,
```

Comment:
No spaces around `=` in an argument default. Also why does `test`'s default value change?